app/views/users/show.html.erb

```diff
@@ -1 +1,37 @@
-<h1>Here is user information for a given user</h1>
\ No newline at end of file
+<section class='main_section'>
+  <div class='user_card'>
+    <div class='user_img'>
+      <img src= <%= @user.photo %> alt= <%= @user.name %> />
+    </div>
+
+    <div class='user_stuff'>
+      <p class='user_name'><%= @user.name %></h2>
+      <p class='posts_no'>Number of Posts: <%= @user.postscounter || 0 %></p>
+    </div>
+  </div>
+ 
+  <div class='bio'>
+    <h4 class='bio_header'>Bio</h4>
+    <p class='bio_text'><%= @user.bio %></p>
+  </div>
+
+  <ul class='posts_list'>
+    <% if @user.posts.empty? %>
+      <li class='post_card'><%= @user.name %> has no posts</li>
+    <% end %>
+
+    <% @user.recent_posts.each_with_index do |post, index| %>
+      <li class='post_card'>
+        <h4 class='post_header'>Post #<%= index + 1 %></h4>
```

Comment:
- [ ] The post title should be a link that allows see the comments of the post, please 🙏🏽 fix that issue to see the comments 👇🏽 

```ruby
<h4 class='post_header'><%= link_to post.title, user_post_path(post) %></h4>
```

Reply:
Done